lib/DataObject/ClassificationstoreDataMarshaller/Multiselect.php

```diff
@@ -34,7 +34,7 @@ public function marshal(mixed $value, array $params = []): mixed
 
     public function unmarshal(mixed $value, array $params = []): mixed
     {
-        if (is_array($value) && strlen($value['value']) > 0) {
+        if (!empty($value['value']) && is_string($value['value'])) {
```

Comment:
if `$value['value']` is a empty string, the explode will generate `['']`. I think it is enough to add `?? ''`.
```suggestion
        if (is_array($value) && strlen($value['value'] ?? '') > 0) {
```

Reply:
Ok. I added a commit to follow your suggestion. Thx.

Reply:
@aMH-techsigns You have revert your change completly and doesn't commit my suggestion. Why not simply use the apply suggestion button of Github?

Reply:
@blankse Oh that was not my intension sorry. I will fix it!
Never used the button before but i will do it that way right now.

Reply:
I think its done. Thx for getting back to me so quick!